tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+	"version": "2.0.3",
+	"compilerOptions": {
+		"declaration": false,
+		"module": "umd",
+		"noImplicitAny": true,
+		"noImplicitThis": true,
```

Comment:
The others I'm thinking of are `noUnusedParameters` and `noUnusedLocals` , but I'm not sure if we've standardized on using those across the Dojo 2 packages? cc @agubler